include/qinterface.hpp

```diff
@@ -32,6 +32,7 @@ unsigned char* qrack_alloc(size_t ucharCount);
 void mul2x2(complex* left, complex* right, complex* out);
 void exp2x2(complex* matrix2x2, complex* outMatrix2x2);
 void log2x2(complex* matrix2x2, complex* outMatrix2x2);
+real1 _norm_helper(complex c);
```

Comment:
Doesn't follow any other existing naming convention.

Reply:
It's purely a helper. This is not meant to be used as part of the public API.

Reply:
Changed to a static protected QInterface method, (like the others I just noticed,) and snake changed to camel case.